### ISSUE TITLE
Cannot enable orderbook checksum when sequence audit is enabled

### ISSUE THREAD

**Issue by dutu** [bitfinexcom/bitfinex-api-node#521]
#### Issue type
- [x] bug
- [ ] missing functionality
- [ ] performance
- [ ] feature request

#### Brief description
If the sequence audit is enabled in constructor (`{ seqAudit: true }`), the orderbook checksum cannot be enabled (`ws.enableFlag(WSv2.flags.CHECKSUM)`)

The following code sequence does not enable the orderbook checksum:
```js
const ws = new WSv2({ apiKey: 'apiKey', apiSecret: 'apiSecret', seqAudit: true, manageOrderBooks: true })
ws.on('open', () => {
  ws.enableFlag(WSv2.flags.CHECKSUM)
  ws.subscribeOrderBook('tBTCUSD', 'P0', '25')
})
ws.open()
```
However, this code sequence (where `seqAudit` is set to `false` in the constructor) enables the orderbook checksum:

```js
const ws = new WSv2({ apiKey: 'apiKey', apiSecret: 'apiSecret', seqAudit: false, manageOrderBooks: true })
ws.on('open', () => {
  ws.enableFlag(WSv2.flags.CHECKSUM)
  ws.subscribeOrderBook('tBTCUSD', 'P0', '25')
})
ws.open()
```
The issue is because the client sends the two flag configuration commands one after each other, and the second `enableFlag` command disables the first flag.
This happens because `this._enabledFlags` is updated only when the server responds. Therefore when sending the second `enableFlag` right after the first (before the server responds), the first flag gets set to `false`.

https://github.com/bitfinexcom/bitfinex-api-node/blob/abbd3c1825aadaf37cc049366fc2c029e6c0c493/lib/transports/ws2.js#L1384-L1391

As consequence the orderbook checksum flag cannot be set if `{ seqAudit: true }` is set in the constructor.

#### Steps to reproduce
```js
const ws = new WSv2({ apiKey: 'apiKey', apiSecret: 'apiSecret', seqAudit: true, manageOrderBooks: true })
ws.on('open', () => {
  ws.enableFlag(WSv2.flags.CHECKSUM)
  ws.subscribeOrderBook('tBTCUSD', 'P0', '25')
})
ws.open()
```

##### Additional Notes:
-

**Comment by dutu** [comment]
As a workaround solution the sequence audit flag should be enabled together with the orderbook checksum flag, rather than in the constructor.

```js
    const ws = new WSv2({ apiKey: 'apiKey', apiSecret:'apiSecret', seqAudit: false, manageOrderBooks: true })
    ws.on('open', () => {
      ws.enableFlag(WSv2.flags.SEQ_ALL | WSv2.flags.CHECKSUM)
      ws.subscribeOrderBook(`tBTCUSD`, 'P0', '25')
    })
```

To eliminate any possible issues, and as a permanent solution I would suggest:
* eliminate the possibility to set `seqAudit: true` in the constructor (to be enabled by `enableFlag()` only)
or
* include the possibility to also set the flag `obChecksum: true` in the constructor

**TL;DR set both flags at once**

**Comment by f3rno** [comment]
Thanks for pointing this out, I've submitted a fix here: https://github.com/bitfinexcom/bitfinex-api-node/pull/522 It will be merged shortly
